### PR TITLE
Mk remove const

### PR DESCRIPTION
**Pull request by michaelckelly**

### CODE REVIEW

File: test/PlaidClientTest.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ describe('plaid.Client', () => {
       plaid.environments.sandbox);
   });
 
-  describe('constructor', () => {
+  describe('varructor', () => {
```
Comment (pbernasconi) [inline test/PlaidClientTest.js]: bad replace here

File: lib/PlaidClient.js [hunk 2]
```diff
@@ -1,9 +1,7 @@
-'use strict';
```
Comment (pbernasconi) [inline lib/PlaidClient.js]: I think we want to leave `use strict` in here?

**Comment by pbernasconi** [comment]
💎 after addressing my two comments!

File: test/PlaidClientTest.js (hunk 1 above)
Comment (michaelckelly) [inline test/PlaidClientTest.js]: ⚡️

File: lib/PlaidClient.js (hunk 2 above)
Comment (michaelckelly) [inline lib/PlaidClient.js]: 👍

**Pull request merged**